woocommerce/storefront
# Update dev dependencies to latest minor version

Updates all dependencies to latest minor version.

Also moves [browserslist](https://github.com/browserslist/browserslist) config out to conventional `.browserslistrc` file; this fixes a warning generated by postcss during build: 

```
  Replace Autoprefixer browsers option to Browserslist config.
  Use browserslist key in package.json or .browserslistrc file.

  Using browsers option cause some error. Browserslist config 
  can be used for Babel, Autoprefixer, postcss-normalize and other tools.

  If you really need to use option, rename it to overrideBrowserslist.

  Learn more at:
  https://github.com/browserslist/browserslist#readme
  https://twitter.com/browserslist
```

This PR is intended as a replacement for #1198, but only updating to latest minor versions. #1198 updates all packages to latest, including major versions. We can look at more aggressive dependency updates in future; perhaps we should configure [Renovate](https://renovate.whitesourcesoftware.com) or similar to help with this.

### How to test this PR

- Clone this branch in your dev environment.
- Reinstall all dependencies – `npm ci`.
- Test Storefront styling, pages and core functionality – e.g. customizer, product pages, shop page, cart and checkout. Everything should work correctly (same as master/last release).

Note: I have compared newly-built `style.css` with the [last release](https://github.com/woocommerce/storefront/releases/tag/version%2F2.5.3) to confirm that [autoprefixer](https://github.com/postcss/autoprefixer) (which uses browserslist) is working correctly. Feel free to independently check this :)
